crime-applications-adaptor/build.gradle

```diff
@@ -70,6 +70,8 @@ dependencies {
 
 	implementation "javax.validation:validation-api:$versions.validationapiVersion"
 
+	implementation 'io.micrometer:micrometer-registry-prometheus'
```

Comment:
Super pedantic I know, but can we use double quoted string here for consistency please